client/src/commands/run.ts

```diff
@@ -26,54 +28,62 @@ function getCode(outputHtml: boolean, selected = false): string {
 }
 
 async function _run(selected = false) {
-  const outputHtml = !!workspace
-    .getConfiguration("SAS")
-    .get("session.outputHtml");
-  const code = getCode(outputHtml, selected);
+  if (loading) return;
+  loading = true;
```

Comment:
the reason for adding loading variable:
at the begining of opening the vscode file, commands.executeCommand will be executed slowly, so the run button will be visible for a while even if be clicked and the run function would be called multiple times.

Reply:
Thanks for explaining.
How about name it "running" rather than "loading"?